src/constants/networks.ts

```diff
@@ -71,7 +71,7 @@ export const NETWORKS_INFO: { [key in ChainId]: NETWORK_INFO } = {
     dmmSwapUrl: 'https://kyberswap.com/',
     factoryAddress: '0x833e4083B7ae46CeA85695c4f7ed25CDAd8886dE',
```

Comment:
please help remove all `factoryAddress` as we dont need anymore

Reply:
sure